src/authors/scss/_body.scss

```diff
@@ -0,0 +1,68 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body{
+ 
+    font-family: $family-font sans-serif;
+     
+     
+      background-color: rgb(51, 51, 88);
+      h1{
+        text-align: center;
+        margin: 30px;
+        font-size: 100px;
+
+      }
+      .wrap{
+        display: flex;
+        border-radius: 0 0 40px 40px;
+        margin: 30px;
+        padding:30px;
+        width: 100%;
+        align-items: center;
+        justify-content: center;
+
+        .box{
+            border: 2px solid black;
+                background-color: lightblue;
+                font-size: 1.5rem;
+              width:150px;
+              height:200px;
+                text-align: center;
+                border-radius: 3px;
+                box-shadow: 0 1.5rem 4rem rgba(0, 0, 0, 0.15);
+                transition: transform .3s; 
+                margin:20px;
+                position: relative;
+
+                .box:hover{
+                    transform: translateY(-1.5rem) scale(1.03); 
```

Comment:
Why `rem` and your own hover effects? Look at the design next time. If it doesn't have any hover effect then it doesn't. Of course, you can always suggest it but make it in such way, that everyone in project are aware and give them chance to say what do they think about it.